Majasdarbs/04-majasdarbs/OlegsKozlovs/Portfolio/css/normalize.css

```diff
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,700;1,700&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    width: 100vw;
+    min-height: 100vh;
+    font-family: 'Montserrat', sans-serif;
+    color: black;
+}
+
+main {
+    background-image: url(../img/background_dt.jpg);
+    background-size: cover;
```

Comment:
you could probably always add
background-repeat: no-repeat;
so it 100% doesn't repeat on larger screens, on 4k for example